game/game.py

```diff
@@ -0,0 +1,25 @@
+import pyglet
+from pyglet.gl import glEnable
```

Comment:
When importing multiple items, you can do a comma-separated list.  It's also common practice to only import modules instead of stuff from modules

```suggestion
from pyglet.gl import glEnable, glBlendFunc, GL_BLEND, GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPH
```

```suggestion
import pyglet.gl as pgl
```

Without a defined style guide though these are fine as is.